maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderController.java

```diff
@@ -90,6 +97,17 @@ public ResponseEntity<RepOrderDTO> updateApplicationDateCompleted(@RequestBody U
         return ResponseEntity.ok(repOrderService.updateDateCompleted(updateAppDateCompleted));
     }
 
+    @GetMapping(value = "/rep-order-usn/{usn}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Retrieve rep order records by USN")
+    @ApiResponse(responseCode = "200",
+            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE)
+    )
+    @StandardApiResponse
+    public ResponseEntity<Object> findByUsn(@PathVariable int usn) {
```

Comment:
If I have understood it correctly, it will return a list of associated MAAT Ids.
So the response type should probably be `Set<Long>`, or maybe `List<Long>`?

Reply:
The response entity still has a useful status field which are used on the functional test end hence wrapped in a ResponseEntity.

Reply:
discussed, happy for you to make a call on this one 👍